src/main/java/lambdaperson/FilteredPersons.java

```diff
@@ -0,0 +1,23 @@
+package lambdaperson;
+
+import personlist.Person;
+import java.util.Comparator;
+import java.util.LinkedHashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class FilteredPersons {
+
+    public static Set<Person> filterPersons(Set<Person> personList) {
```

Comment:
A better way is to define a Service class, like `FilterPersonsService` which will have an instance method inside. 
Try to avoid writing static utility methods - they make code organizing hard and usually break OOP principles